lib/unifex/specs_dsl.ex

```diff
@@ -235,6 +235,10 @@ defmodule Unifex.Specs.DSL do
 
       type my_enum :: :option_one | :option_two | :option_three | ...
 
+  Enum constants can be given an explicit value with `enum_value`
+
+      type my_explicit_enum :: enum_value(:option_one, 1) | :option_two | :option_three | ...
```

Comment:
Please mention, that second argument passed to `enum_value/2` can be used only in the code in C/C++ while Elixir API still supports only atoms